package.json

```diff
@@ -9,7 +9,8 @@
 		"build:src": "rimraf lib && npx tsc -p tsconfig.dist.json",
 		"build:spec": "rimraf \"./spec/*.!(ts)\" && npx tsc -p tsconfig.spec.json",
 		"prepublishOnly": "npm run build",
-		"test": "npm run build:spec && npx jasmine"
+		"test": "npm run build:spec && npx jasmine",
+		"prepare": "npm run build"
```

Comment:
Not so clear why you integrated this script 🤔 @billytrend

Reply:
I'm not sure if I needed it or not but I was trying to make this work 
https://github.com/run-pass/run-pass.github.io/blob/4ea73b90c485b462e06a450b9aa582fdf7754087/cloudflare-worker/package.json#L27

Reply:
Okay I understand why you needed it. [According to NPM](https://docs.npmjs.com/cli/v8/using-npm/scripts), `prepare` is run on these situations:

> - Runs any time before the package is packed, i.e. during npm publish and npm pack
> - Runs BEFORE the package is packed
> - Runs BEFORE the package is published
> - Runs on local npm install without any arguments
> - Run AFTER prepublish, but BEFORE prepublishOnly

And, since this is a Typescript project, the thing falls under the 4th case (or under the 3rd, if you start counting from 0 😄).

In the real world (not specific fixes), we don't need it to run build on all these cases. So you can remove it. 😉
You can create a different branch in which you include this fix **and** the compiled version, so the lib folder. Then, you can install the dependency from a specific branch. By doing this way, you keep the fix branch clear to be merged. 👍

Reply:
yeah exactly. Alright, i've reverted the change! ty